src/applications/disability-benefits/526EZ/config/form.js

```diff
@@ -119,8 +119,8 @@ const initialData = {
 const formConfig = {
   urlPrefix: '/',
   intentToFileUrl: '/evss_claims/intent_to_file/compensation',
-  // submitUrl: `${environment.API_URL}/v0/21-526EZ`,
-  submit: () => Promise.resolve({ attributes: { confirmationNumber: '123123123' } }),
+  submitUrl: `${environment.API_URL}/v0/disability_compensation_form/submit`,
+  // submit: () => Promise.resolve({ attributes: { confirmationNumber: '123123123' } }),
```

Comment:
We can remove this comment now if we're able to use the actual submit endpoint.

Reply:
Yeah, I was going to leave it as a sort of reminder if we have to do some data translations. And now that I think about it, I'm sure we will since we've got basically all the things wrapped up in the `disabilities` array. :thinking:

Reply:
Sounds good, could you please set up a ticket with your thoughts for that?

Reply:
New ticket for this can be found at https://github.com/department-of-veterans-affairs/vets.gov-team/issues/10875